en/flight_controller/beaglebone_blue.md

```diff
@@ -205,12 +205,13 @@ After acquiring the pre-built library,
 Run the following commands on the BeagleBone Blue (i.e. via SSH):
 
 1. Install dependencies:
+
    ```sh
    sudo apt-get update
-   sudo apt-get install cmake python-empy
+   sudo apt-get install cmake python3-empy=3.3.4_2
    ```
-2. Clone the PX4 Firmware directly onto the BeagleBone Blue.
-3. Continue with the [standard build system installation](../dev_setup/dev_env_linux.md).
+1. Clone the PX4 Firmware directly onto the BeagleBone Blue.
```

Comment:
FYI Markdown doesn't care - if you put `1.` for all it autonumbers correctly. I tend to do it this way because if you have to move steps later and reorder, you don't have to go through and correct lines that haven't actually changed.